CHANGELOG.rst

```diff
@@ -1,5 +1,17 @@
 .. _changelog:
 
+0.27.0
+------
+
+Renku ``0.27.0`` upgrades the Keycloak version that is shipped with the project.
+
+Internal Changes
+~~~~~~~~~~~~~~~~
+
+**Improvements**
+
+- **YAT**: (BREAKING CHANGE) Use a new Keycloak Helm chart (codecentric/keycloakx) and upgrade Keycloak to 20.0.3. This requires modifying your current values file to work with the new Keycloak Helm chart, see (`the helm chart values changelog <https://github.com/SwissDataScienceCenter/renku/blob/master/helm-chart/values.yaml.changelog.md>`_) for instructions.
```

Comment:
Don't use "YAT" in public changelogs, it's an internal name

Reply:
We usually use  Renku Core Service

Reply:
In this case it should just be something like "supporting services" or "3rd party services" or "Infrastructure" - "core service" is actually a service we write

Reply:
done